ebs/ebs.go

```diff
@@ -706,7 +705,7 @@ func (d *Driver) MountVolume(req Request) (string, error) {
 		var validID = regexp.MustCompile(`. output mount: special device /dev/([a-z]+) does not exist`)
 		if validID.MatchString(errorStr) {
 			if delErr := util.ObjectDelete(volume); delErr != nil {
-				log.Warnf("Deleting object=%q in response to an error while unmounting produced: %s", volume, delErr)
+				log.Warnf("Deleting volume=%v in response to an error while mounting produced another error: %s", volume, delErr)
```

Comment:
Deleting object is different to deleting volume, in that the prior means the reference to the volume in Convoy's state is being cleaned, not the volume itself.